docs/faststore/docs/customization/building-sections/override-use-cases/adding-more-icon-options-to-the-alert-component.mdx

```diff
@@ -0,0 +1,125 @@
+2. In the `CustomIconsAlert` folder, create the `CustomIconsAlert.tsx` file.
+3. Import the following to the `CustomIconsAlert.tsx` file:
+
+        ```tsx
+        import { getOverriddenSection } from '@faststore/core'
+        ```
+
+4. Define the `CustomIconsAlert` component:
+
+        ```tsx focus=3:7
+        import { AlertSection, getOverriddenSection } from '@faststore/core'
+
+        const CustomIconsAlert = getOverriddenSection({
+        Section: AlertSection,
+        })
+
+        export default CustomIconsAlert
+        ```
+
+    - This is an Alert component override with custom icon options in the Headless CMS. The component is defined using `getOverriddenSection`.
+    - This code changes only the schema to include more options for native Icons supported by `@faststore/ui`.
+    - The override options specify:
+      - **`Section`:** The section to be overridden (Alert).
+
+    > ℹ️ For further details on code implementation, see the [`CustomIconsAlert`](https://github.com/vtex-sites/playground.store/tree/main/src/components/sections/CustomIconsAlert) folder available in the [playground.store](https://github.com/vtex-sites/playground.store) repository.
+
+### Synchronizing the changes with the Headless CMS
+
+Add the section to the Headless CMS by following the instructions available in [syncing components to the Headless CMS](https://developers.vtex.com/docs/guides/faststore/overrides-syncing-components-with-the-headless-cms). The following schema was used as an example:
+
+        ```json
+        [    
+            {
+            "name": "CustomIconsAlert",
+            "schema": {
+                "title": "Alert with more Icon options",
+                "description": "Add an alert that has more Icon options",
+                "type": "object",
+                "required": ["icon", "content", "dismissible"],
+                "properties": {
+                "icon": {
+                    "type": "string",
+                    "title": "Icon",
+                    "enumNames": [
+                    "Bell",
+                    "BellRinging",
+                    "Checked",
+                    "Info",
+                    "Truck",
+                    "User",
+                    "Plus",
+                    "PlusCircle",
+                    "Ruler",
+                    "Star",
+                    "Storefront",
+                    "Twitter"
+                    ],
+                    "enum": [
+                    "Bell",
+                    "BellRinging",
+                    "Checked",
+                    "Info",
+                    "Truck",
+                    "User",
+                    "Plus",
+                    "PlusCircle",
+                    "Ruler",
+                    "Star",
+                    "Storefront",
+                    "Twitter"
+                    ]
+                  }
+                }
+              }
+            }
+        ]
+        ```
```

Comment:
- should not be indented
- where should this schema be used?

Reply:
I added the location on where it should be used ([`cms/faststore/sections.json`](https://github.com/vtex-sites/playground.store/blob/main/cms/faststore/sections.json#L58-L101)) on [this line](https://github.com/vtexdocs/dev-portal-content/pull/1264/files#diff-acad65f586b9bcb070fb9beda1b3c19ea97558aa2dd61d2c339b382ea4cec62aR108).